pick.lic

```diff
@@ -246,43 +246,56 @@ class LockPicker
       return
     end
 
-    if @make_pets
-      case bput("put my #{box} in my #{@settings.picking_pet_box_source}", 'You put your', "You just can\'t get", "There isn\'t any more")
-      when "You just can\'t get", "There isn\'t any more"
-        bput("put my #{box} in my #{@settings.picking_box_source}", 'You put your')
-        @pet_count = @pet_goal
-      when 'You put your'
-        @pet_count += 1
+    unless @settings.use_skeleton_key
+      if @make_pets
+        case bput("put my #{box} in my #{@settings.picking_pet_box_source}", 'You put your', "You just can\'t get", "There isn\'t any more")
+        when "You just can\'t get", "There isn\'t any more"
+          bput("put my #{box} in my #{@settings.picking_box_source}", 'You put your')
+          @pet_count = @pet_goal
+        when 'You put your'
+          @pet_count += 1
+        end
+        return
       end
-      return
-    end
 
-    if DRStats.thief?
-      if DRC.bput("glance my #{box}", /there.*(no|\d) traps? left/, 'I could').scan(/there.*(no|\d) traps? left/)[0].first.to_i > 0
-        DRC.message("Hit the Thief Hasten bug, disarming again!") if UserVars.lockpick_debug
-        attempt_open(box)
+      if DRStats.thief?
+        if DRC.bput("glance my #{box}", /there.*(no|\d) traps? left/, 'I could').scan(/there.*(no|\d) traps? left/)[0].first.to_i > 0
+          DRC.message("Hit the Thief Hasten bug, disarming again!") if UserVars.lockpick_debug
+          attempt_open(box)
+        else
+          attempt_pick(box)
+        end
       else
         attempt_pick(box)
       end
-    else
-      attempt_pick(box)
```

Comment:
The only change to these lines is they are now indented under new line 249, `unless @settings.use_skeleton_key`